### PR TITLE
SOFTWARE-3694 Add stash-cache tests to the container builds

### PR DESCRIPTION
**Pull request by efajardo**

### CODE REVIEW

File: travis/stashcache-cache-config/10-stash-cache.conf [hunk 1]
```diff
@@ -0,0 +1,4 @@
+[program:stash-cache]
```
Comment (brianhlin) [inline travis/stashcache-cache-config/10-stash-cache.conf]: Why do we have to add this file? Isn't this already included in the container image?

File: travis/stashcache-travis-ci-test/testfile [hunk 2]
```diff
@@ -0,0 +1 @@
+This is a test file for travis ci stashcache docker test
```
Comment (brianhlin) [inline travis/stashcache-travis-ci-test/testfile]: Could you add a new line to the end here?

File: travis/stashcache-cache-config/10-stash-cache.conf (hunk 1 above)
Comment (efajardo) [reply travis/stashcache-cache-config/10-stash-cache.conf]: Yes but it it includes the authenticated and I did not want to get the results of errors of supervisrod.

Comment (brianhlin) [reply travis/stashcache-cache-config/10-stash-cache.conf]: Ah ok, could you add a comment to the supervisord file that you're adding explaining the reasoning?

Comment (efajardo) [reply travis/stashcache-cache-config/10-stash-cache.conf]: I managed to get it working removing this.

**Pull request merged**